Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:14-alpine
+
+WORKDIR /grabtern-frontend
+
+COPY ./package*.json /grabtern-frontend
+
+RUN npm install
+
+COPY . /grabtern-frontend
+
+CMD ["npm","start"]
```

Comment:
we need multistage build wrna aise toh file size boht bdi hogi

Reply:
Sure will do it.